apps/fluent-tester/src/FluentTester/testPages.ios.ts

```diff
@@ -35,26 +35,26 @@ export const tests: TestDescription[] = [
     component: NativeButtonTest,
     testPage: HOMEPAGE_NATIVEBUTTON_BUTTON,
   },
-  {
-    name: 'Callout Test',
-    component: CalloutTest,
-    testPage: HOMEPAGE_CALLOUT_BUTTON,
-  },
-  {
-    name: 'ContextualMenu Test',
-    component: ContextualMenuTest,
-    testPage: HOMEPAGE_CONTEXTUALMENU_BUTTON,
-  },
+  // {
+  //   name: 'Callout Test',
+  //   component: CalloutTest,
+  //   testPage: HOMEPAGE_CALLOUT_BUTTON,
+  // },
+  // {
+  //   name: 'ContextualMenu Test',
+  //   component: ContextualMenuTest,
+  //   testPage: HOMEPAGE_CONTEXTUALMENU_BUTTON,
+  // },
```

Comment:
We have a preference to not have commented out code in public repos, so it would be preferred to just delete the code altogether. There should be a list of all the tests in just `testPages.ts` that is easy enough to reference to remember what tests exists.

Reply:
similar to macOS test app should we do disable tableviewcell look? or should we really delete?

Reply:
A disabled button for tests that don't work on a platform would be a good idea, but a more complex change. I think that could be a good followup.
I think right now between the components / experimental components, there isn't really one platform that actually has every test so it's fine to just delete them for now.

Reply:
Should I also delete the import statements as well since they aren't being used? I'm not sure what the tableviewcell look is.

Reply:
If you don't delete import statements, I think you'll get an `unused import` error in the PR checks (probably what's failing under PR (Build, test, and bundle Javascript) right now)., so definitely remove them.

Sure, I think it's no harm to add those two tests to `testPages.ts`. At some point I do think we should consolidate to just have one `testPages` file, but I still feel that's outside the scope of this simple change.